components/automate-ui/src/app/modules/infra-proxy/edit-environment-attribute-modal/edit-environment-attribute-modal.component.ts

```diff
@@ -105,9 +105,9 @@ export class EditEnvironmentAttributeModalComponent implements OnChanges, OnInit
       this.cookbookConstraints.forEach((element) => {
         this.selectedCookbookNames.push(element.name);
       });
-      this.selectedCookbookNames.forEach((ele) => {
-        this.constraintKeys.forEach((element, index) => {
-          if (ele === element) {
+      this.selectedCookbookNames.forEach((cookbookName) => {
+        this.constraintKeys.forEach((key, index) => {
+          if (cookbookName === key) {
```

Comment:
Author note: code clarity -- much easier to comprehend with different variable names.